internal/sdkgen/sdkgen.go

```diff
@@ -78,12 +82,18 @@ func Generate(ctx context.Context, customerID, workspaceID, lang, schemaPath, he
 		return err
 	}
 
-	if errs := g.Generate(context.Background(), schema, schemaPath, lang, outDir, isRemote, compile); len(errs) > 0 {
-		for _, err := range errs {
-			logger.Error("", zap.Error(err))
-		}
+	err = events.Telemetry(ctx, shared.InteractionTypeTargetGenerate, func(ctx context.Context, event *shared.CliEvent) error {
```

Comment:
Feels like we shouldn't need this here _and_ in the command wrapper? The way I would envision this working is we add a field to the command wrapper interface like `EventType` that then gets automatically added to executions in `model/command.go`

Reply:
Resolved in slack 👍